src/ajax.ts

```diff
@@ -37,7 +52,31 @@ function getConnector() {
  * @returns {Promise<any>}
  */
 export function send(url: string, data : any = {}, method = 'get', expectedDataType = 'json'): Promise<any> {
-  return getConnector().then((c) => c.send(url, data, method, expectedDataType));
+  if (method === 'get' || method === 'head') {
+    data = encodeParams(data); //encode in url
+    if (data) {
+      url += (/\?/.test(url) ? '&' : '?') + data;
+    }
+    data = null;
+  }
+  const options :any = {
```

Comment:
I have to manipulate the `objects` later on. If I would remove the `any`, I wouldn't be able to add new fields, since TypeScript infers the datatype from the assignment